views/partials/formatjs.hbs

```diff
@@ -20,3 +20,11 @@
     HandlebarsIntl.registerWith(Handlebars);
 </script>
 {{/if}}
+
+{{#if usesEmberIntl}}
+<script src="/vendor/handlebars/handlebars{{min}}.js"></script>
+<script src="/vendor/jquery/jquery{{min}}.js"></script>
+<script src="/vendor/ember/ember{{min}}.js"></script>
+<script src="/vendor/ember-intl/ember-intl{{min}}.js"></script>
+<script src="/vendor/formatjs/ember-intl-locale-data.js"></script>
+{{/if}}
```

Comment:
I think we need an initialization script here like some of the other integrations are doing.